t/220-mvcc.t

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env tarantool
+local qc = require('queue.compat')
+local log   = require('log')
+if not qc.check_version({2, 6, 1}) then
+    log.info('Tests skipped, tarantool version < 2.6.1')
+    return
+end
+local yaml  = require('yaml')
```

Comment:
As far as I can see, you are not using yaml below.

Reply:
deleted